airgun/entities/host_new.py

```diff
@@ -32,6 +38,163 @@ def get_details(self, entity_name, widget_names=None):
         self.browser.plugin.ensure_page_safe()
         return view.read(widget_names=widget_names)
 
+    def edit_system_purpose(
+        self, entity_name, role=None, sla=None, usage=None, release_ver=None, add_ons=None
+    ):
+        """
+        Function that edits the system purpose of a host
+
+        Args:
+            entity_name: Name of the host
+            role: Role to be assigned
+            sla: SLA to be assigned
+            usage: Usage to be assigned
+            release_ver: Release version to be assigned
+            add_ons: Add-ons to be assigned
+
+        Raises:
+            ValueError: If no parameters are passed.
+        """
+
+        if not any([role, sla, usage, release_ver, add_ons]):
+            raise ValueError(
+                'At least one of the role, sla, usage, release_ver, add_ons ' 'must be provided!'
```

Comment:
Is it intended to have two strings here, or formatting glitch?

Reply:
Formating glitch will look into that.